terraform/modules/front_door/waf.tf

```diff
@@ -50,6 +50,11 @@ resource "aws_wafv2_web_acl" "this" {
         name        = "AWSManagedRulesCommonRuleSet"
         vendor_name = "AWS"
         rule_action_override {
+          # This rule blocks request bodies over 8KB in size, but CORE needs file uploads so we remove this restriction

```

Comment:
Thought this would be useful. Delta had a similar comment

Reply:
We could increase the limit to 64KB but this is not something Delta did. Depends on how much we want to look at file upload requests (presuming I've understood this correctly - that such requests are more risky because we can't check as much of the request body, and other request types will probably be below 16KB )

Reply:
Yeah. Again, we'll want to check some realistic uses (but that can happen later).

Reply:
Added a note about this in functionality regression testing ticket [CLDC-2738](https://dluhcdigital.atlassian.net/browse/CLDC-2738)